ppr-ui/src/components/parties/PartySearch.vue

```diff
@@ -85,7 +85,7 @@ export default defineComponent({
   setup (props, context) {
     const localState = reactive({
       searchValue: '',
-      autoCompleteResults: [],
+      autoCompleteResults: null,
```

Comment:
will this throw an error since you check *autoCompleteResults.length* inside party-autocomplete?

Reply:
I popped in the null check